components/server/src/setup.rs

```diff
@@ -34,12 +35,22 @@ macro_rules! fatal {
 // TODO: There is a very small chance that duplicate files will be generated if there are
 // a lot of logs written in a very short time. Consider rename the rotated file with a version
 // number while rotate by size.
+//
+// The file name format after rotated is as follows: tikv-2021-12-21T00-19-38.378.log
 fn rename_by_timestamp(path: &Path) -> io::Result<PathBuf> {
-    let mut new_path = path.to_path_buf().into_os_string();
-    new_path.push(format!(
-        ".{}",
-        Local::now().format(logger::DATETIME_ROTATE_SUFFIX)
-    ));
+    let parent = path.parent().unwrap().as_os_str();
+    let prefix = path.file_stem().unwrap();
+    let mut new_path = OsString::from(parent);
+    new_path.push("/");
+    new_path.push(prefix);
+    let dt = Local::now().to_rfc3339_opts(SecondsFormat::Millis, false);
+    // remove zone
+    let (dt, _) = dt.split_at(dt.len() - 6);
+    new_path.push(format!("-{}", dt.replace(":", "-")));
```

Comment:
What about `Local::now().format("%Y-%m-%dT%H-%M-%S%.3f")`?

Reply:
Changed to this, thanks!